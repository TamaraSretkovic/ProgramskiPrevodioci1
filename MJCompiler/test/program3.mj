//	Oporavak od greske iskaz dodele

program test302
    int aa, c;
{
	void main() int b; 123;{
		aa = 5 + 6;
	}
}

