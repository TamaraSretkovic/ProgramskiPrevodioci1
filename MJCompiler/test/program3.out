INFO  01:15:53,879 - Compiling source file: /home/tamarica/Documents/tamara/velika skola/8 semestar/PP1/projekat/projecat_2019_2020/MJCompiler/test/program3.mj
INFO  01:15:53,902 - #2 program
INFO  01:15:53,903 - #46 test302
INFO  01:15:53,903 - #46 int
INFO  01:15:53,903 - #46 aa
INFO  01:15:53,904 - #6 ,
INFO  01:15:53,904 - #46 c
INFO  01:15:53,904 - #5 ;
INFO  01:15:53,904 - #3 {
INFO  01:15:53,905 - #10 void
INFO  01:15:53,905 - #46 main
INFO  01:15:53,905 - #12 (
INFO  01:15:53,905 - #11 )
INFO  01:15:53,906 - #46 int
INFO  01:15:53,906 - #46 b
INFO  01:15:53,907 - #5 ;
INFO  01:15:53,907 - #47 123
ERROR 01:15:53,907 - 
Sintaksna greska na liniji 6
INFO  01:15:53,907 - #5 ;
INFO  01:15:53,907 - #3 {
INFO  01:15:53,907 - #46 aa
INFO  01:15:53,908 - #23 =
ERROR 01:15:53,908 - Uspesan oporavak od greske na liniji 6 pri deklarisanju varijable do , ili ;
INFO  01:15:53,910 - #47 5
INFO  01:15:53,910 - #32 +
INFO  01:15:53,911 - #47 6
INFO  01:15:53,911 - #5 ;
INFO  01:15:53,911 - #4 }
INFO  01:15:53,912 - #4 }
ERROR 01:15:53,913 - Parsiranje NIJE uspesno zavrseno!
